features/round-profile-pictures/data.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Round Profile Pictures",
+  "description": "All profile pictures on the Scratch website will be rounded.",
+  "credits": [
+    { "username": "Scratchfangs", "url": "https://scratch.mit.edu/users/scratchfangs/" },
+    { "username": "rgantzos", "url": "https://scratch.mit.edu/users/rgantzos/" }
+  ],
+  "type": ["Website", "Theme"],
+  "tags": [],
+  "scripts": [{ "file": "script.js", "runOn": "/*" }],
+  "options": [
+    { "id": "Rounding Percentage", "name": "Rounding Percentage", "default": "50%" }
```

Comment:
If we're going to add things like this, then we should also add an option validation system.

Reply:
I'll make an issue for this and it should be added in the next few days.

Reply:
@Wilamaxin It's been merged, something like `"validation":[{ "regex": "^(100(?:\\.0{1,2})?|\\d{1,2}(?:\\.\\d{1,2})?)%$", "explanation": "Percentage must be between 0% and 100%."}]` for the option would work now.